### PR TITLE
App cleanup

### PR DESCRIPTION
**Pull request by lauradobson**
1. Added more description to the description of "Open/Closed" Status
2. Cosmetic changes on the courses table on the application page.
3. Larger/more noticeable "reason" button
4. Student ID and other errors showing up in the Basic Information table. Student form validation occurs prior to the form trying to submit.
5. Updated Application instructions directly on master.
6. ExpecTAtions dropdown created - initialized to "No".

### CODE REVIEW

**Comment by smwatts** [comment]
@lauradobson mind merging master into your branch? I'll test it after :)

**Review by amyleblond — CHANGES_REQUESTED** [review]
i'm still not getting the student_id error check
screen shot 2018-03-04 at 10 29 15 pm - just redirects me to the page but doesnt say why

also the popup message is showing on the screen under submit before thepopup loads, not sure if this was intended--ran incognito and did all the collectstatic
screen shot 2018-03-04 at 10 36 02 pm

File: taworks/taform/templates/taform/application.html [hunk 1]
```diff
@@ -172,7 +173,7 @@ <h2>Course Application</h2>
     </div>
 {% for i in courses %}
 <div class="required">
-    <div id="row">
+    <div class="row">
         <div id="left">
```
Comment (j2kan) [inline taworks/taform/templates/taform/application.html]: can you change line 177 and line 178 to classes as well? they should have never been ids.

File: taworks/taform/views.py [hunk 2]
```diff
@@ -201,14 +201,16 @@ def apply(request):
...
         context = {
                 's_form' : s_form,
                 'courses' : models.Course.objects.all(),
                 'app_form' : a_forms,
                 'error' : "Error: The student ID must be 8 characters.",
                 'AC' : AC,
                 'app_status' : app_status,
-                'status_date': status_date
+                'status_date': status_date,
+                'form_valid' : form_valid
```
Comment (j2kan) [inline taworks/taform/views.py]: if form isn't valid here, does it need to go in the try statement?

**Review by amyleblond — APPROVED** [review]

File: taworks/taform/views.py [hunk 3]
```diff
@@ -224,14 +226,16 @@ def apply(request):
                     app.save()
                     course_number += 1
             else:
+                form_valid = True
```
Comment (j2kan) [inline taworks/taform/views.py]: wouldn't the form also not be valid if it's in the else statement?

**Comment by j2kan** [comment]
form_valid seems to always be false now?

**Comment by smwatts** [comment]
Mind merging master again? sorry we're developing too fast for what we're reviewing lol. Just think a few of my comments will be based on things fixed in earlier merged PRs.

Let me know if thats the case for these comments:

Can this text go above the submit button?
screen shot 2018-03-05 at 11 16 59 pm

This damn asterisk lol, should be removed after the citizenship selection:
screen shot 2018-03-05 at 11 14 58 pm

**Comment by j2kan** [comment]
does enter reason button still disappear?

**Review by j2kan — CHANGES_REQUESTED** [review]
form_valid is always false. what's the intention with it?

File: taworks/taform/templates/taform/application.html [hunk 4]
```diff
@@ -191,18 +192,18 @@ <h2>Course Application</h2>
...
 
 <h2>Confirmation</h2>
-<div id="divWordWrap">
+<div class="divWordWrap">
     I fully understand that I can only submit one application per academic term, 
     and that once the application is submitted, no changes or modifications of 
     any sort can be done to my application.
 </div><br>
-<div id="divWordWrap">
+<div class="divWordWrap">
```
Comment (amyleblond) [inline taworks/taform/templates/taform/application.html:200]: can we make this word wrap longer? confirmation looks weird
screen shot 2018-03-06 at 11 43 17 am

**Pull request closed without merging**